finesse/hardware/temperature/tc4820.py

```diff
@@ -155,8 +156,8 @@ def temperature(self) -> Decimal:
 
     @property
     def power(self) -> int:
-        """The current power output of the device."""
-        return self.request_int("020000")
+        """The current power output of the device, as a percentage of maximum."""
+        return int(self.request_int("020000") * 100 / TC4820_MAX_POWER)
```

Comment:
I think we should just return a float instead -- you can `round()` it for the progress bar.

Reply:
Return a `Decimal` via
```python
return self.request_decimal("020000") * 100 / TC4820_MAX_POWER
```
or still request an int but return the value of the percentage conversion as a float?